FWCore/Framework/src/StreamSchedule.cc

```diff
@@ -573,24 +578,46 @@ namespace edm {
           auto aliasedToModuleLabels = info.getParameterNames();
           for (auto const& mod : aliasedToModuleLabels) {
             if (not mod.empty() and mod[0] != '@' and conditionalmods.find(mod) != conditionalmods.end()) {
-              auto aliasPSet = proc_pset.getParameter<edm::ParameterSet>(mod);
```

Comment:
Because the `ParameterSet` was asked from the process `ParameterSet` (while it should have been asked from the `EDAlias`' `ParameterSet`), which actually contains a `ParameterSet` with the alias-for EDModule label (the module's `ParameterSet`).